audit/jsonformatter.go

```diff
@@ -0,0 +1,27 @@
+package audit
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	. "github.com/sirupsen/logrus"
+)
+
+type JsonFormatter struct {
```

Comment:
Curious, logrus has it's own JSONFormatter, why did you implement a new one? I'm reasonably sure I know the reason, just checking!

Reply:
Yeah so the comment I added here kind of alludes to it but its not 100% clear. But if i were to use the default json formatter we would end up with those fields as well. Honestly I did it because I felt like they didn't provide value and in the other more custom cases (csv and text) I wasn't going to include them either because they dont make a ton of sense in those outputs so I wanted to make it consistent across all of them at least. 

This does point out that we are kind of sort of abusing logrus to do this output. It's not bad but since we are ignoring fields its not standard if you will.

```
// Note this doesn't include Time, Level and Message which are available on
// the Entry. Consult `godoc` on information about those fields or read the
// source of the official loggers.
```